js-src/Trustmark.spec.ts

```diff
@@ -49,8 +45,8 @@ describe("Trustmark", () => {
   let testImageHeight: number;
 
   const trustmarkConfig: TrustmarkConfig = {
-    variant: "B" as TrustmarkVariant, // Original Trustmark model
-    version: "BCH_3" as TrustmarkVersion, // Tolerates 3 bit flips
+    variant: "B", // Original Trustmark model
```

Comment:
Do you still need the type import if you don't convert here? For other parts of the file, I guess?

Reply:
Only for invalid variant test.